todo-app/src/components/TodoList.js

```diff
@@ -0,0 +1,32 @@
+import React, { useReducer } from "react";
+// import { initialState, reducer } from "../reducers/reducer";
+
+export default function TodoList(props) {
+  //   const [state, dispatch] = useReducer(reducer, initialState);
```

Comment:
Looks like you moved the useReducer hook to a higher component instead of calling it twice in TodoList and TodoForm.
Very smart refactor!